source/files/pearson/pearson – kópia.json

```diff
@@ -0,0 +1,90 @@
+[{
```

Comment:
wait, why do we have a file with "kopia" in its name?

Reply:
It's a backup file in case the original get's rewritten unwantedly, so that I can revert using this backup. It took me a lot of time to edit those json files to get the right pictures there.